packages/integration/src/lib/map/map-tools/map-tools.component.scss

```diff
@@ -29,6 +29,6 @@ mat-tab-group ::ng-deep .mat-tab-body-wrapper {
 }
 
 :host ::ng-deep  .mat-tab-body-content  {
-    overflow: hidden;
+    overflow: auto;
```

Comment:
Maybe validate if the map-legend tool is also fixed.

Reply:
Please, find a more generic method based on igo-layer-list-legend

Reply:
<div> has been added for eliminate a composant igo-layer-legend-list